test/torchaudio_unittest/transforms/transforms_test.py

```diff
@@ -163,6 +163,45 @@ def test_mfcc(self):
 
         self.assertTrue(torch_mfcc_norm_none.allclose(norm_check))
 
+    def test_lfcc(self):
+        audio_orig = self.waveform.clone()
+        audio_scaled = self.scale(audio_orig)  # (1, 16000)
```

Comment:
Instead of `self.waveform` and `self.scale`, please use `common_utils.get_whitenoise` or something similar.
We are trying to remove these as they create unnecessary couplings among tests and cause confusion to developers.

Reply:
Addressed in 40ddf05.